src/Compilers/CSharp/Portable/Symbols/SymbolWithAnnotations.cs

```diff
@@ -744,7 +804,7 @@ public override TypeSymbol TypeSymbol
                 {
                     if ((object)_resolved == null)
                     {
-                        if (_underlying.IsReferenceType && ((CSharpParseOptions)_nullableTypeSyntax.SyntaxTree.Options).IsFeatureEnabled(MessageID.IDS_FeatureStaticNullChecking))
+                        if (!_underlying.IsValueType && ((CSharpParseOptions)_nullableTypeSyntax.SyntaxTree.Options).IsFeatureEnabled(MessageID.IDS_FeatureStaticNullChecking))
```

Comment:
Not related to this PR: I didn't understand why the feature being on or off affects the symbol.
If `_underlying` is a reference type, but the feature is not enabled, we'll go ahead and make a `Nullable<SomeRefType>` which seems illegal (`T` is constrained to `struct` in `Nullable<T>`).